webpack.config.js

```diff
@@ -142,13 +145,14 @@ const webpackConfig = {
 				return outputPath;
 			},
 		} ),
-		new CopyWebpackPlugin(
-			wcAdminPackages.map( ( packageName ) => ( {
+		new CopyWebpackPlugin({
+
+			patterns: wcAdminPackages.map( ( packageName ) => ( {
 				from: `./packages/${ packageName }/build-style/*.css`,
-				to: `./${ packageName }/`,
-				flatten: true,
-				transform: ( content ) => content,
+				to: `./${ packageName }/[name][ext]`,
+				noErrorOnMissing: true
```

Comment:
Curious, why do we need to set `noErrorOnMissing` to `true` here?

Reply:
Some packages don't have any css at all, so when this pattern match tries to run against such a folder it errors out